testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -204,6 +213,10 @@ impl EventObserver {
         self.send_payload(payload, PATH_MEMPOOL_TX_SUBMIT);
     }
 
+    fn send_dropped_mempool_txs(&self, payload: &serde_json::Value) {
+        self.send_payload(payload, PATH_MEMPOOL_TX_DROP);
```

Comment:
Note this sends a lot of `StaleGarbageCollect` payloads without any transactions. Not a blocker just a tiny inefficiency.

Reply:
Noted -- will add a check to make sure its only sending payloads when they're non-empty.